operator/docker/user_setup

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -x
+
+# ensure $HOME exists and is accessible by group 0 (we don't know what the runtime UID will be)
+echo "${USER_NAME}:x:${USER_UID}:0:${USER_NAME} user:${HOME}:/sbin/nologin" >> /etc/passwd
```

Comment:
nologin isn't going to work for all cases. There is a (VERY) long debate about this in the distroless repo where `nonroot` was added.